projects/dashboards/src/widgets/conditional/conditional-widget-renderer.component.ts

```diff
@@ -8,23 +8,15 @@ import { ConditionalModel } from './conditional.model';
 @Renderer({ modelClass: ConditionalModel })
 @Component({
   selector: 'ht-conditional-widget',
-  styleUrls: ['./conditional-widget-renderer.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
-  template: `
-    <div class="conditional-widget">
-      <ng-container [hdaDashboardModel]="this.getModel() | async"> </ng-container>
-    </div>
-  `
+  template: ` <ng-container [hdaDashboardModel]="this.getChildModel() | async"> </ng-container> `
 })
 export class ConditionalWidgetRendererComponent extends WidgetRenderer<ConditionalModel> {
-  @ViewChild('containerContent', { read: ViewContainerRef, static: true })
-  public container!: ViewContainerRef;
-
-  public getModel(): Observable<ModelJson> {
-    return this.model.getModel().pipe(takeUntil(this.destroyed$));
+  public getChildModel(): Observable<object> {
+    return this.model.getChildModel().pipe(takeUntil(this.destroyed$));
```

Comment:
nit: this takeUntil isn't a problem but not necessary ( we used to use these a lot more if you were looking at old code - now we try instead where possible to push the subscriptions into the DOM which will auto unsubscribe.

Reply:
Will fix locally. Thanks.